wordpress-mobile/WordPress-iOS
# Jetpack migration flow - add success card

Fixes #19612

This PR adds the migration success card to My Site Home (iPhone only) and My Site menu.

iPhone Home | iPhone Menu | iPad
-- | -- | --
<img height=450 src=https://user-images.githubusercontent.com/34376330/203443015-89f7c51e-2335-45ca-823a-6bdb3422077a.png> | <img height=450 src=https://user-images.githubusercontent.com/34376330/203443027-ce69f233-4237-4912-ac05-fe1c473cf7b0.png> | <img height=450 src=https://user-images.githubusercontent.com/34376330/203596273-1315bd32-e4f0-4357-af93-ba8b06fec367.png>


To test:

- checkout this branch
- find the following property in `MigrationSuccessCardView`
    `private static let showCard = true`
- set it to `true`. **NOTE: this property is temporary and has been put there for the sole purpose of testing the card. The exact condition to determine wether to show the card will be added in a future PR.**
- Build/run the Jetpack target
- Navigate to My Site, if not already there, and make sure that the card shows up in "Menu", and in "Home" (only on iPhone)

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
